config/sites/classrooms.uiowa.edu/views.view.room_list.yml

```
@@ -0,0 +1,1086 @@
+            label: 'Technology features'
+            description: ''
+            use_operator: false
+            operator: field_room_technology_features_target_id_op
+            operator_limit_selection: false
+            operator_list: {  }
+            identifier: field_room_technology_features_target_id
+            required: false
+            remember: false
+            multiple: false
+            remember_roles:
+              authenticated: authenticated
+              anonymous: '0'
+              viewer: '0'
+              editor: '0'
+              publisher: '0'
+              webmaster: '0'
+              administrator: '0'
+            reduce: false
+          is_grouped: false
+          group_info:
+            label: ''
+            description: ''
+            identifier: ''
+            optional: true
+            widget: select
+            multiple: false
+            remember: false
+            default_group: All
+            default_group_multiple: {  }
+            group_items: {  }
+          reduce_duplicates: false
+          vid: technology_features
+          type: select
+          hierarchy: false
+          limit: true
+          error_message: true
+          save_lineage: false
+          force_deepest: false
+          parent: null
+          level_labels: ''
+          hierarchy_depth: 0
+          required_depth: 0
+          none_label: '- Please select -'
+        field_room_max_occupancy_value:
+          id: field_room_max_occupancy_value
+          table: node__field_room_max_occupancy
+          field: field_room_max_occupancy_value
+          relationship: none
+          group_type: group
+          admin_label: ''
+          plugin_id: numeric
+          operator: '='
+          value:
+            min: ''
+            max: ''
+            value: ''
+          group: 1
+          exposed: true
+          expose:
+            operator_id: field_room_max_occupancy_value_op
+            label: 'Max Occupancy (field_room_max_occupancy)'
+            description: null
+            use_operator: false
+            operator: field_room_max_occupancy_value_op
+            operator_limit_selection: false
+            operator_list: {  }
+            identifier: field_room_max_occupancy_value
+            required: false
+            remember: false
+            multiple: false
+            remember_roles:
+              authenticated: authenticated
+            min_placeholder: null
+            max_placeholder: null
+            placeholder: null
+          is_grouped: true
+          group_info:
+            label: 'Max Occupancy'
+            description: ''
+            identifier: field_room_max_occupancy_value
+            optional: true
+            widget: select
+            multiple: false
+            remember: false
+            default_group: All
+            default_group_multiple: {  }
+            group_items:
+              1:
+                title: '25 or less'
```

Tiny nit if you don't mind 😬 

```suggestion
                title: '25 or fewer'
```